styles/style.css

```diff
@@ -0,0 +1,96 @@
+body {
+  margin: 0;
+  padding: 0;
+  width: 100vw;
+  overflow: hidden;
+}
+
+/* Start of header styles */
+header {
+  height: 72px;
+  color: white;
+  display: flex;
+  align-items: center;
+  padding: 0 20px;
+  font-size: 18px;
+  font-family: Inter, sans-serif;
+  justify-content: space-between;
+  background-color: #1c1a19;
+}
+
+#bars > svg {
+  color: white;
+  width: 25px;
+  height: 25px;
+}
+
+#bars > svg:hover {
+  color: yellow;
+}
+
+/* Start of headline-section styles */
+.headline-section {
+  width: 100%;
+  height: 100vh;
+  margin-top: -20px;
+  overflow: hidden;
+  display: flex;
+  background-image: url("/public/images/headline-section.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.sider-icons-menu {
+  width: 80px;
+  height: 100%;
+  padding: 0 20px;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: 18px;
+  background-color: #1c1a19;
+}
+
+.sider-icons-menu > span {
+  padding: 15px 0;
+}
+
+.headline-text-contents {
+  width: auto;
+  margin-left: 1px;
+  padding: 0 10px;
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY css you should combine duplicate properties. Example, you can create class `.d-flex { display: flex}` and use this class in html file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Reply:
Well done